sdk/batch/batch-rest/samples/v1-beta/typescript/src/quick-start.ts

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Azure Batch quickstart sample
+ */
+
+import createClient, {
+  BatchClient,
+  CreateJobParameters,
+  CreatePoolParameters,
+  CreateTaskParameters,
+  isUnexpected,
+  paginate,
+} from "@azure-rest/batch";
+import { AzureNamedKeyCredential } from "@azure/core-auth";
+
+// Load the .env file if it exists
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const batchAccountEndpoint = process.env["BATCH_ACCOUNT_ENDPOINT"] || "<batch account endpoint>";
+const batchAccountName = process.env["BATCH_ACCOUNT_NAME"] || "<batch account name>";
+const batchAccountKey = process.env["BATCH_ACCOUNT_KEY"] || "<batch account key>";
+const poolName = "samplepool";
+const jobId = "samplejob2";
+
+async function main() {
+  const credentials = new AzureNamedKeyCredential(batchAccountName, batchAccountKey);
```

Comment:
I believe Dave said we should have customers move away from using NamedKeyCredentials and instead use DefaultAzureCredential https://learn.microsoft.com/en-us/javascript/api/@azure/identity/defaultazurecredential?view=azure-node-latest